### PR TITLE
Update example to configure pods from configmap

### LINKED ISSUE #36650: Issue with k8s.io/docs/concepts/configuration/configmap/
> ```
>   volumes:
>     # You set volumes at the Pod level, then mount them into containers inside that Pod
>     - name: config
> ```

indenting is wrong on `- name`

### PR DESCRIPTION
**Pull request by Rishit-dagli**
This PR works on:

- update the example to configure pods from the configmap
- Use the `codenew` shortcode instead of directly code

Closes #36650

### CODE REVIEW

File: content/en/examples/configmap/configure-pod.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  - name: config
+    configMap:
+      # Provide the name of the ConfigMap you want to mount.
+      name: game-demo
+      # An array of keys from the ConfigMap to create as files
+      items:
+      - key: "game.properties"
+        path: "game.properties"
+      - key: "user-interface.properties"
+        path: "user-interface.properties"
```
Comment (shivaylamba) [inline content/en/examples/configmap/configure-pod.yaml]: Hi @Rishit-dagli can you add an extra end of file line here

File: content/en/examples/configmap/configure-pod.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
```
Comment (shivaylamba) [inline content/en/examples/configmap/configure-pod.yaml:39]: Hi Rishit can you add an end of file line here

**Review by sftim — COMMENTED** [review]
We shouldn't commit `.hugo_build.lock`

/hold
Please omit that file @Rishit-dagli, **and rebase** (we can help if you're not sure how to)

**Comment by Rishit-dagli** [comment]
> We shouldn't commit .hugo_build.lock

Ah, sorry for missing that

**Review by sftim — COMMENTED** [review]
Thanks

/kind cleanup
/lgtm
/approve

**Pull request merged**